pages/share-target.vue

```diff
@@ -8,7 +8,7 @@ definePageMeta({
 
 useWebShareTarget()
 
-const pwaIsInstalled = process.server ? false : useNuxtApp().$pwa.isInstalled
+const pwaIsInstalled = process.client && !!useNuxtApp().$pwa?.isInstalled
```

Comment:
the middleware runs before? if so, you can replace it with `useNuxtApp().$pwa.isInstalled`